rali/rali_pybind/amd/rali/plugin/pytorch.py

```diff
@@ -81,13 +95,50 @@ def __next__(self):
         else:
             self.loader.copyToTensorNHWC(self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
         
-        self.loader.getImageLabels(self.labels)
-        self.labels_tensor = torch.from_numpy(self.labels).type(torch.LongTensor)
-    
-        if self.tensor_dtype == types.FLOAT:
-            return torch.from_numpy(self.out), self.labels_tensor
-        elif self.tensor_dtype == types.FLOAT16:
-            return torch.from_numpy(self.out.astype(np.float16)), self.labels_tensor
+        if((self.loader._name == "Caffe2ReaderDetection") or (self.loader._name == "CaffeReaderDetection")):
+            sum = 0
```

Comment:
Codacy found an issue: [Redefining built-in 'sum'](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)